### ISSUE TITLE
All Things Open conference

### ISSUE THREAD

**Issue by mazzystr** [kubevirt/community#95]
All Things Open 2021

ATO Call for Papers - [link](https://2021.allthingsopen.org/call-for-papers/) 

ATO Talk Topics - [link](https://2021.allthingsopen.org/talk-topics-of-interest/)

- [ ] Develop proposal to present at ATO2021 @stu-gott 
- [ ]

**Comment by kwiesmueller** [comment]
I'd be happy to help with this. My own single-node VM for personal stuff is running K3s with K3os right now and I'm working on getting kubevirt up (need to get my provider enable kvm for me).

**Comment by mazzystr** [comment]
@kwiesmueller @stu-gott Maybe we can add a _hybrid cloud_ element to the demo??

**Comment by kwiesmueller** [comment]
I thought about that for a bit as well, if the event ends up remote (or even if it doesn't) and we have more rPis at various locations we could connect them up. Could count as kind of an Edge demo even. 
So see if we can add nodes from multiple presenters or as you suggested add additional small nodes from other clouds.

**Comment by cwilkers** [comment]
I have a reasonably stable 1Gb connection over fibre in Houston, TX, and would like to add an RPI4 to my setup.

**Comment by davidvossel** [comment]
> Demonstrate running mixed workloads.

so, will this be a cluster with both x84 and arm nodes?

**Comment by mazzystr** [comment]
Absolutely.  I have a 4 core Intel i5, 16 gig mem, 256 GiB ssd HP EliteDesk
I can throw at the demo.

**Comment by mazzystr** [comment]
I spoke with NASA last night.  OpenStack Ceilometer continuously explodes on them, dashboards are difficult to build.

Their #1 ask for this demo is node exporters for virtual machines sending metrics to prometheus with a grafana dashboard.

**Comment by mazzystr** [comment]
Presentation form is complete

**Comment by mazzystr** [comment]
I ordered an Argon One system

**Comment by mazzystr** [comment]
Good news!  Our paper has been accepted!  We will be presenting at ATO 2021!

**Comment by mazzystr** [comment]
Intro video added to RH ATO 2021 slide deck

Veronica Cooley is engaged to order swag (stickers) for physical table
